site/features.md

```diff
@@ -118,6 +118,22 @@ IP addresses of external services the hosts or containers need to
 connect to. The same IP range must be used everywhere, and the
 individual IP addresses must, of course, be unique.
 
+If you restart a container, it will retain the same IP addresses on
+the weave network:
```

Comment:
Do you think we should mention it because if a system is severely overloaded it could take longer than 5 seconds to do the restart?  If we believe that will impact users, we should perhaps go for a longer timeout.